src/routes/resources/[resourceContentId]/+page.svelte

```diff
@@ -732,7 +732,7 @@
                                     {machineTranslationStore}
                                 />
                                 {#if mediaType === MediaTypeEnum.text}
-                                    <div class="flex h-10 flex-row items-center text-sm text-gray-500">
+                                    <div class="flex h-10 flex-row items-center px-[0.75rem] text-sm text-gray-500">
```

Comment:
Nit, looks like `px-3` should be equivalent

Reply:
I can use `px-3` I just based it off of the daisy ui applied styles in the inspector.